portfolio/src/main/java/com/google/sps/servlets/RandomQuote.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches a random quote from the server and adds it to the DOM.
+ */
+function getRandomQuote() {
+  console.log('Fetching a random quote.');
```

Comment:
Why do you want to log this to console?

Reply:
I must have left it there from when I was debugging thank you, I will remove now